src/App.js

```diff
@@ -79,42 +79,49 @@ function App() {
   return (
     <div className="App">
       <header>
-        <nav>
-          <ul>
-          <li><a href="">Home</a></li>
-          <li><a href="">About</a></li>
-          <li><a href="">Calendar</a></li>
-          </ul>
-        </nav>
-        <h1>one day at a time</h1>
+        <div className="wrapper">
+          <h1>one day at a time</h1>
+        </div>
       </header>
 
-      <section className="about-section">
-        <h2>How does it work?</h2>
-        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Esse libero dolores adipisci quidem totam nisi voluptas, beatae dolor, nulla porro id odio ullam voluptatum eos provident vero reiciendis quis labore?</p>
-      </section>
+      <section className="about-and-submit">
+        <div className="about-section">
+          <div className="wrapper">
+          <h2>How does it work?</h2>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Esse libero dolores adipisci quidem totam nisi voluptas, beatae dolor, nulla porro id odio ullam voluptatum eos provident vero reiciendis quis labore?</p>
```

Comment:
Hey I can read Latin, and that is absolutely disgusting. Take that back about my grandmother, very unprofessional!!